lib/node_modules/@stdlib/utils/delay/README.md

```diff
@@ -0,0 +1,206 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2022 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# delay
+
+> Invoke a function after a specified number of milliseconds.
+
+<!-- Section to include introductory text. Make sure to keep an empty line after the intro `section` element and another before the `/section` close. -->
+
+<section class="intro">
+
+</section>
+
+<!-- /.intro -->
+
+<!-- Package usage documentation. -->
+
+<section class="usage">
+
+## Usage
+
+```javascript
+var delay = require( '@stdlib/utils/delay' );
+```
+
+#### delay( fcn, wait\[, ...args] )
```

Comment:
Discussed in internal TODOs (see https://github.com/stdlib-js/todo/blob/f8da702717c99f31a6eb21820875eb31ae08f881/todos/2.md#L1679), a suggestion was made for allowing human readable duration strings.

The motivation for this suggestion is that working in milliseconds is not "natural" or the most intuitive. In contrast, we could support something like [`1d2h1m30s10ms`](https://docs.rs/duration-string/latest/duration_string/) or use a duration string `HH:MM:SS` which could be extended to ISO 8601 [durations](https://en.wikipedia.org/wiki/ISO_8601#Durations).

Adding support for duration strings would provide entail additional packages for parsing the duration strings. The former format is probably easier and maps more readily to user expectations. For ISO 8601 duration strings, one must specify milliseconds as fractional seconds.

Even if we adopted the former now, ISO 8601 support could be added later, as the two formats do not have overlapping symbols (the former above uses lowercase, while ISO 8601 uses uppercase and/or colons).

Reply:
In the same internal TODO list, there is an item for `parseDuration` for handling the former duration format.

Reply:
Should be able to move forward with supporting duration strings once https://github.com/stdlib-js/stdlib/pull/561 is merged.